TRANSLATIONS.rst

```diff
@@ -0,0 +1,145 @@
+🐍 🌍 🚀 Translation Release Strategy
+======================================
+
+Thanks for your interest in translating the micro:bit Python Editor.
+Translations for all the Micro:bit Educational Foundation’s editors and
+apps are contributed by an amazing team of volunteers and we’d love for
+you to be a part of this too.
+
+This document sets out the strategy and process for how the Foundation
+manages the translation of the Python Editor hosted at
+python.microbit.org
+
+1. Translation and Management in the Python Editor
+--------------------------------------------------
+
+Translation is the act of taking the written materials in one language
+and converting them into another language in the most meaningful way
+possible. In terms of the micro:bit Python Editor, translation happens
+on the `strings marked for translation in the projects'
+codebase <https://github.com/bbcmicrobit/PythonEditor/lang/en.js>`__. The default language of
+the editor is English and you may select another available language
+using the Globe icon in the main menu.
+
+1.1 Crowdin
++++++++++++
+The Micro:bit Educational Foundation use the `Crowdin
+platform <https://crowdin.com/project/microbitorg>`__ to manage
+translations to manage and support community translations for it’s web
```

Comment:
maybe `to manage translations, to organise and to support community translations` ?

Reply:
I think that's repetition, so `to manage and support community translations`